bin/add_frontend_server.rb

```diff
@@ -178,26 +186,14 @@ def execute_ssh_on_channel(channel, command, exit_error_string)
   f.write(JSON.dump(configuration_json))
 end
 
-Net::SCP.upload!('gateway.code.org', username, '/tmp/new_knife_config.json', '/tmp/new_knife_config.json')
-puts 'New configuration file written.'
+Net::SCP.upload!('gateway.code.org', username, '/tmp/new_knife_config.json', "/tmp/new_knife_config#{file_suffix}.json")
+puts 'New configuration file uploaded, now loading it.'
 
 Net::SSH.start('gateway.code.org', username) do |ssh|
```

Comment:
That sounds reasonable.  Is there a way for the user to determine the suffix?  

Related thought, do we want to keep a log file that captures what happened with the deploy?